src/mscorlib/src/System/Reflection/Runtime/Assemblies/AssemblyNameHelpers.cs

```diff
@@ -0,0 +1,156 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+/*============================================================
+**
+  Type:  AssemblyNameHelpers
+**
+==============================================================*/
+
+using System;
+using System.Globalization;
+using System.IO;
+using System.Text;
+using System.Collections.Generic;
+
+namespace System.Reflection.Runtime.Assemblies
```

Comment:
We should move these to the System.Reflection namespace. (Ideally that should have been done last year when these files were moved out of Reflection.Core.)